### PR TITLE
Added members_feedback table

### PR DESCRIPTION
**Pull request by SimonBackx**
fixes https://github.com/TryGhost/Team/issues/2041

### CODE REVIEW

**Review by lenabaidakova — APPROVED** [review]
🚀

File: ghost/core/core/server/data/migrations/versions/5.19/2022-10-10-10-04-add-members-feedback-table.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const {addTable} = require('../../utils');
+
+module.exports = addTable('members_feedback', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    type: {type: 'string', maxlength: 50, nullable: false, defaultTo: 'positive', validations: {isIn: [['positive', 'negative']]}},
```
Comment (allouis) [inline ghost/core/core/server/data/migrations/versions/5.19/2022-10-10-10-04-add-members-feedback-table.js:5]: Just FYI the `validations` config is not used at the DB level but at the model level, so it's not necessary here.

**Review by allouis — COMMENTED** [review]
Looks good - only thing is, do we want to edit these records, or have a new event for each vote?

**Comment by SimonBackx** [comment]
> Looks good - only thing is, do we want to edit these records, or have a new event for each vote?

I think changing a vote is an edge case (and would only happen right after voting when a user makes a mistake) and it would make things a lot easier if we would not have to take that into account and just edit a vote when they change a vote instead of creating a new one.

**Review by daniellockyer — APPROVED** [review]

**Pull request merged**